go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
```

Comment:
Consider removing the unnecessary version constraints for indirect dependencies. This can help reduce complexity in your module file since Go modules will pull the latest compatible versions automatically when needed.